src/syscheckd/db/include/fimDBHelper.hpp

```diff
@@ -161,16 +162,18 @@ namespace FIMDBHelper
         update["table"] = tableName;
         update["data"] = {item};
```

Comment:
this could be replaced with `update["data"] = nlohmann::json::array({item});`  to make it more explicit?